plugins/text-editor-resources/src/components/extensions.ts

```diff
@@ -141,14 +141,29 @@ export function inlineCommandsConfig (
 ): Partial<CompletionOptions> {
   return {
     suggestion: {
-      items: () => {
-        return [
+      items: ({ query }: { query: string }) => {
+        const items = [
           { id: 'image', label: textEditor.string.Image, icon: view.icon.Image },
           { id: 'table', label: textEditor.string.Table, icon: view.icon.Table2 },
           { id: 'code-block', label: textEditor.string.CodeBlock, icon: view.icon.CodeBlock },
           { id: 'separator-line', label: textEditor.string.SeparatorLine, icon: view.icon.SeparatorLine },
           { id: 'todo-list', label: textEditor.string.TodoList, icon: view.icon.TodoList }
         ].filter(({ id }) => !excludedCommands.includes(id as InlineCommandId))
+
+        // to handle case of `todo-list` and `action-item` being the same
+        const searchableItems = items.map(item =>
+          item.id === 'todo-list'
+            ? { ...item, searchLabels: ['action-item', textEditor.string.TodoList] }
```

Comment:
Using internal text keys as a search condition won't work well. Users will expect the search to work for what they see in the menu so for the interface in english it may poorly work in some cases (where the text doesn't match the key) and for the interface in other languages it won't work at all. Please consider searching by translated texts.